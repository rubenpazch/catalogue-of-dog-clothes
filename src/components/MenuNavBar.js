import React from 'react';
import Navbar from 'react-bootstrap/NavBar';
import Nav from 'react-bootstrap/Nav';
import { Col, Row } from 'react-bootstrap';
import styles from '../css/menunavbar.module.css';

function MenuNavBar() {
  return (
    <Row className={styles.wrapper}>
      <Col className={styles.leftwrapper} md={2} />
      <Col className={styles.midlewrapper} md={7}>
        <ul className={styles.wrapperMenu}>
          <li href="#home" className={styles.itemList}>
						<a>Home</a>
					</li>
          <li href="#features" className={styles.itemList}>
						<a> Features</a>
					</li>
          <li href="#pricing" className={styles.itemList}>
            <a>Pricing</a>
          </li>
        </ul>
      </Col>
      <Col className={styles.rigthwrapper} md={3}>
        <p>cart</p>
      </Col>
    </Row>
  );
}

export default MenuNavBar;
