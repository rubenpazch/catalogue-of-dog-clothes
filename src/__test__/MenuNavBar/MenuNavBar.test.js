import React from 'react';
import ReactDOM from 'react-dom';
import {
  BrowserRouter as Router,
  Route,
} from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import MenuNavBar from '../../components/MenuNavBar';

afterEach(cleanup);

it('renders MenuNavBar without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Router>
      <Route>
        <MenuNavBar />
      </Route>
    </Router>,
    div,
  );
});

it('renders item correctly', () => {
  const { getByTestId } = render(
    <Router>
      <Route>
        <MenuNavBar />
      </Route>
    </Router>,
  );
  getByTestId('MenuNavBar');
});

it('render ItemTour information', () => {
  const { getByTestId } = render(
    <Router>
      <Route>
        <MenuNavBar />
      </Route>
    </Router>,
  );
  expect(getByTestId('MenuNavBar')).toHaveTextContent('ADVENTURES');
  expect(getByTestId('MenuNavBar')).toHaveTextContent('MUSEUMS');
});

it('machets snapshot MenuNavBar', () => {
  const itemTour = renderer.create(
    <Router>
      <Route>
        <MenuNavBar />
      </Route>
    </Router>,
  ).toJSON();
  expect(itemTour).toMatchSnapshot();
});
