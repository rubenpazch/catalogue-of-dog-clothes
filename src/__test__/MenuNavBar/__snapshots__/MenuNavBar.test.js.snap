// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`machets snapshot MenuNavBar 1`] = `
<div
  className="wrapper row"
  data-testid="MenuNavBar"
>
  <div
    className="leftwrapper col-md-2"
  />
  <div
    className="midlewrapper col-md-7"
  >
    <ul
      className="wrapperMenu"
    >
      <li
        className="itemList"
        href="#home"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
      </li>
      <li
        className="itemList"
        href="#features"
      >
        <a
          href="www.google.com"
        >
           ADVENTURES
        </a>
      </li>
      <li
        className="itemList"
        href="#pricing"
      >
        <a
          href="www.google.com"
        >
          MUSEUMS
        </a>
      </li>
      <li
        className="itemList"
        href="#pricing"
      >
        <a
          href="www.google.com"
        >
          LUXURY
        </a>
      </li>
      <li
        className="itemList"
        href="#pricing"
      >
        <a
          href="www.google.com"
        >
          TRADITIONAL
        </a>
      </li>
    </ul>
  </div>
  <div
    className="rigthwrapper col-md-3"
  >
    <p>
      -
    </p>
  </div>
</div>
`;
